configs/templates/ufs-srw-dev/spack.yaml

```diff
@@ -0,0 +1,50 @@
+spack:
+  config:
+    install_tree:
+      root: $env/install
+    deprecated: true
+  modules:
+    default:
+      roots:
+        lmod: $env/install/modulefiles
+        tcl: $env/install/modulefiles
+      lmod:
+        projections:
+          parallelio: 'pio/{version}'
```

Comment:
I believe it is correct that lmod doesn't have the mpi/compiler dependencies, can you confirm @kgerheiser ?